Gifski/IntTextField.swift

```diff
@@ -0,0 +1,61 @@
+import AppKit
+import Carbon
+
+final class IntTextField: NSTextField, NSTextFieldDelegate {
+	// Delta used for arrow navigation
+	var delta = 1
+
+	// Delta used for option + arrow navigation
+	var biggerDelta = 10
+
+	var onBlur: ((Int) -> Void)?
+	var onTextDidChange: ((Int) -> Void)?
+
+	required init?(coder: NSCoder) {
+		super.init(coder: coder)
+		setup()
+	}
+
+	override init(frame frameRect: NSRect) {
+		super.init(frame: frameRect)
+		setup()
+	}
+
+	private func setup() {
+		delegate = self
+	}
+
+	override func performKeyEquivalent(with event: NSEvent) -> Bool {
+		let key = Int(event.keyCode)
+		let isHoldingOption = event.modifierFlags.contains(.option)
+
+		let delta: Int
```

Comment:
```swift
let delta = isHoldingOption ? biggerDelta : delta
```

Reply:
it would be nice simplification, but we still need a switch to decide on multiplier so I think we should leave it as is to make the intent clearer (one statement decides the delta value vs multiple lines in and out of switch)

Reply:
Right, I misread the code. I meant:

```swift
override func performKeyEquivalent(with event: NSEvent) -> Bool {
		let key = event.specialKey
		let isHoldingOption = event.modifierFlags.contains(.option)
		let initialDelta = isHoldingOption ? alternativeDelta : delta

		let delta: Int
		switch key {
		case .upArrow?:
			delta = initialDelta
		case .downArrow?:
			delta = initialDelta * -1
		default:
			return super.performKeyEquivalent(with: event)
		}
```